src/App/App.js

```diff
@@ -1,31 +1,41 @@
 import React, { Component } from "react"
 import "./App.css"
 import testData from "../testData"
-
 import AllMoviesView from "../AllMoviesView/AllMoviesView"
 
 class App extends Component {
   constructor() {
     super()
     this.state = {
-      detailViewId: null,
+      movieId: null,
       movies: [...testData]
     }
   }
 
-  displayMovie = id => {
-    console.log(id)
+  displayMovie = (id) => {
+    this.setState({ movieId: id })
+  }
+
+  closeMovie = () => {
+    this.setState({ movieId: null })
   }
 
   render() {
     return (
-      <main>
-        <h1>Rancid Tomatillos</h1>
-        <AllMoviesView 
-          movies={this.state.movies} 
-          displayMovie={this.displayMovie}
-        />
-      </main>
+      <>
```

Comment:
Hey, overall this looks great! Only thing is, did we lose the opening `<main>` tag here?

Reply:
Scratch that. Just saw you moved <header> outside of main